README.md

```diff
@@ -27,6 +27,29 @@ $ ./scripts/build.sh
 ```shell
 $ ./scripts/deploy.sh
 ```
+* Setup the workload identity service account
+```shell
+# Create a service account for workload identity
+$ gcloud iam service-accounts create gke-workload
+
+# Allow "default/mypod" to act as the new service account
+$ gcloud iam service-accounts add-iam-policy-binding \
+    --role roles/iam.workloadIdentityUser \
+    --member "serviceAccount:$PROJECT_ID.svc.id.goog[default/mypod]" \
+    gke-workload@$PROJECT_ID.iam.gserviceaccount.com
+```
+* Create a secret that the workload identity service account can access
+```shell
+# Create a secret with 1 active version
+$ echo "foo" > secret.data
+$ gcloud secrets create test --replication-policy=automatic --data-file=secret.data
```

Comment:
nit: `testsecret` or `mysecret` here is probably more descriptive to help make it crystal clear in all contexts

Reply:
sgtm!